src/screens/TopicPageScreen/index.js

```diff
@@ -51,10 +51,15 @@ const TopicPageScreen = (props) => {
   const [memberCount, setMemberCount] = React.useState(0);
   const [isHeaderHide, setIsHeaderHide] = React.useState(false);
   const opacityHeaderAnimation = React.useRef(new Animated.Value(1)).current;
+  const coverPath = topicDetail?.cover_path || null;
 
   const animatedHeight = React.useRef(
     new Animated.Value(
-      dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT + dimen.size.TOPIC_FEED_HEADER_HEIGHT + normalize(4)
+      (coverPath
+        ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
+        : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT) +
+        dimen.size.TOPIC_FEED_HEADER_HEIGHT +
+        normalize(4)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `coverPath` variable is used to determine the animated height. However, there's no null check for `coverPath` before using it in the ternary operator. If `coverPath` is `null`, it might lead to unexpected behavior.
```diff
- const coverPath = topicDetail?.cover_path || null;
+ const coverPath = topicDetail?.cover_path;

- const animatedHeight = React.useRef(
-   new Animated.Value(
-     (coverPath
-       ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
-       : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT) +
-       dimen.size.TOPIC_FEED_HEADER_HEIGHT +
-       normalize(4)
-   )
- ).current;
+ let animatedHeightValue = dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT;
+ if (coverPath) {
+   animatedHeightValue = dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER;
+ }
+ const animatedHeight = React.useRef(
+   new Animated.Value(
+     animatedHeightValue +
+     dimen.size.TOPIC_FEED_HEADER_HEIGHT +
+     normalize(4)
+   )
+ ).current;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->